### ISSUE TITLE
Multitech MTCAP series Gateways in the list of supported platforms for converting to Helium Gateway.

### ISSUE THREAD

**Issue by javanessian** [helium/gateway-rs#41]
Hi,

I would like to request Multitech MTCAP series Gateways to be supported platforms for converting to a Helium Gateway.

Thank you,
Jay

**Comment by lthiery** [comment]
I'm working on this with Jay and we somehow get the following error:

```
root@mtcap:/var/config/home/admin# opkg install helium-gateway-v1.0.0-alpha.9-mtcdt.ipk

Collected errors:

* Solver encountered 1 problem(s):
* Problem 1/1:
*   - package helium_gateway-1.0.0-alpha.9.mtcdt does not have a compatible architecture
*
* Solution 1:
*   - do not ask to install helium_gateway-1.0.0-alpha.9.mtcdt
```

Yet:
```
#########################################################
#      mPower(TM) Edge Intelligence Conduit AP AEP      #
#########################################################
admin@192.168.0.2's password:
Last login: Mon May 17 10:18:20 2021
admin@mtcap:~$ uname -m
armv5tejl
```

I would think `armv5te-unknown-linux-musleabi` would support this `armv5tejl`, but apparently not...

**Comment by madninja** [comment]
@lthiery so the question is whether `armv5tejl` is the lower common denominator here or if it's completely different? Did you try building a `armv5tejl` binary and running it on the mtcdt?

**Comment by lthiery** [comment]
I could not find another toolchain other than the `armv5te-unknown-linux-musleab` that might work on the `armv5tejl`. I gave the old college try on a few but it wasn't easy because they're not fully supported Rust toolchains. If we really want to support this model, we should ask Multitech to provide more info on what the processor so we can at least know which architecture we're targeting.

In the meantime, I've recommended @javanessian find another gateway that fits his criterion without being this particular model.
